offchain/danaswap-browser/src/DanaSwap/Browser/Home.purs

```diff
@@ -19,18 +24,239 @@ main = HA.runHalogenAff do
+            ]
+        , currentPool: Nothing
+        , activity: Swap
+        }
     , render
     , eval: H.mkEval $ H.defaultEval
+        { handleAction = handleAction
+        }
     }
   where
-  render _ =
-    HH.div_
-      [ HH.text "this is Home"
+  handleAction :: forall slots. Action -> H.HalogenM State Action slots o m Unit
+  handleAction = case _ of
+    ResetCurrentPool -> H.modify_ _ { currentPool = Nothing }
+    SetCurrentPool pool -> H.modify_ _ { currentPool = Just pool }
+    SetActivity activity -> H.modify_ _ { activity = activity }
+
+  render :: forall slots. State -> H.ComponentHTML Action slots m
+  render { pools, currentPool, activity } =
+    HH.section
+      [ mkClass "section my-6 pb-6" ]
+      $
+        [ HH.div
+            [ mkClass "columns is-mobile" ]
+            [ HH.div [ mkClass "pools-table-header column" ]
+                [ HH.div [ mkClass $ "columns" ]
+                    [ HH.div [ mkClass "column has-text-left" ] [ HH.text "Pool" ]
+                    ]
+                ]
+            , HH.div [ mkClass "pools-table-header column" ]
+                [ HH.div [ mkClass $ "columns" ]
+                    [ HH.div [ mkClass "column has-text-centered" ] [ HH.text "Value" ]
+                    ]
+                ]
+            , HH.div [ mkClass "pools-table-header column" ]
+                [ HH.div [ mkClass $ "columns" ]
+                    [ HH.div [ mkClass "column has-text-centered" ] [ HH.text "Fee" ]
+                    ]
+                ]
+            , HH.div [ mkClass "column" ] [ HH.text " " ]
+            ]
+
+        ]
+      <> concat
+        ( renderPool
+            <$> pools
+        )
+    where
+    renderPool pool@(Pool p) =
+      [ HH.div
+          [ mkClass "pools-table-row columns is-mobile is-vcentered p-3 my-3"
+          , HE.onClick $ \_ ->
+              if currentPool == Just pool then
+                ResetCurrentPool
+              else
+                SetCurrentPool pool
+          ]
+
+          [ HH.div [ mkClass "column" ]
+              [ HH.p [ mkClass "is-size-5-desktop is-size-7-touch has-text-left" ] [ HH.text $ show pool ]
+              ]
+          , HH.div [ mkClass "column" ]
+              [ HH.p [ mkClass "is-size-5-desktop is-size-7-touch has-text-centered" ] [ HH.text $ show p.assetA <> " / " <> show p.assetB ]
+              ]
+          , HH.div [ mkClass "column" ]
+              [ HH.p [ mkClass "is-size-5-desktop is-size-7-touch has-text-centered" ] [ HH.text $ show p.fee <> "ADA" ]
+              ]
+          , HH.div [ mkClass "column has-text-centered" ]
+              [ if currentPool == Just pool then mkIcon "angle-up" else mkIcon "angle-down"
+              ]
+          ]
+      ] <> renderPoolDropdown pool
+
+    renderPoolDropdown pool =
+      if currentPool == Just pool then
+        [ HH.div [ mkClass "pools-table-row-dropdown box my-6" ]
```

Comment:
https://www.w3.org/WAI/ARIA/apg/example-index/combobox/combobox-select-only.html

Missing a lot of accessibility

Reply:
indeed, for MVP, I reckon that's not a blocker

Reply:
For testing and prototyping, sure, but not to production